tiledb/sm/misc/test/CMakeLists.txt

```diff
@@ -38,3 +38,11 @@ commence(unit_test misc)
         unit_math.cc
     )
 conclude(unit_test)
+
+commence(unit_test mgc_dict)
```

Comment:
It's certainly a welcome improvement to put the validation tests into Catch2. I didn't even know we had a special case for this.